idseq_dag/steps/run_cdhitdup.py

```diff
@@ -39,7 +55,19 @@ def run(self):
                 args=cdhitdup_params
             )
         )
+        self._upload_clstr_files()
```

Comment:
nit: thoughts on naming this `_include_clstr_files` or similar? since it doesn't actually get uploaded in the func but is flagged for uploading later

Reply:
fair point